src/main/java/com/amazon/aocagent/testamis/A1AmazonLinux.java

```diff
@@ -0,0 +1,25 @@
+package com.amazon.aocagent.testamis;
+
+import com.amazon.aocagent.enums.S3Package;
+
+public class A1AmazonLinux extends LinuxAMI {
+  @Override
+  public String getAMIId() {
+    return "ami-091a6d6d0ed7b35fd";
+  }
+
+  @Override
+  public String getLoginUser() {
+    return "ec2-user";
+  }
+
+  @Override
+  public boolean isARM() {
```

Comment:
maybe we don't need this method, because we can tell the architecture from S3Package.AMAZON_LINUX_ARM64_RPM

Reply:
yeah that also works , I just thought this make code more readable instead of complex string manipulation.